module_voice.lua

```diff
@@ -57,15 +61,39 @@ function Module:OnEnable(guild)
 	local config = self:GetConfig(guild)
 
 	if not config.TriggerChannel then
-		return false, 'No channel configured for private voice.'
+		return false, Bot:Format(guild, 'VOICE_MISCONFIG')
 	end
 
 	local persistentData = self:GetPersistentData(guild)
-
 	if not persistentData.PrivateVoiceChannels then
 		persistentData.PrivateVoiceChannels = {}
+
+		return true
+	end
+
+--  cleanup config and unused channels after reboot
+	for channelId, ownerId in pairs(persistentData.PrivateVoiceChannels) do
+		local channel = guild.voiceChannels:find(
+			function(voice) if voice.id == channelId then return voice end end
+		)
+
+		if channel then
+			local isOwnerConnected = channel.connectedMembers:find(
+				function(member) if member.id == ownerId then return true end end
+			)
```

Comment:
```suggestion
			local isOwnerConnected = channel.connectedMembers:find(
				function(member) return member.id == ownerId end
			)
```
Would work too, if im not mistaken

Reply:
yep